templates/admintools-deployment.yaml

```diff
@@ -28,6 +28,7 @@ spec:
         app.kubernetes.io/component: admintools
         app.kubernetes.io/part-of: {{ .Chart.Name }}
     spec:
+      serviceAccountName: {{ include "temporal.serviceAccountName" . }}
```

Comment:
this should optionally allow setting service account to nil/empty/unset.

Reply:
@nvtkaszpir  this is done - please take a look at it